FFMpegCore/FFMPEG/FFMpeg.cs

```diff
@@ -380,38 +382,54 @@ public VideoInfo ReplaceAudio(VideoInfo source, FileInfo audio, FileInfo output,
                 new OutputArgument(output)
             ));
         }
-        
+
         public VideoInfo Convert(ArgumentContainer arguments, bool skipExistsCheck = false)
         {
             var (sources, output) = GetInputOutput(arguments);
-            _totalTime = TimeSpan.FromSeconds(sources.Sum(source => source.Duration.TotalSeconds));
+            if (sources != null)
+                _totalTime = TimeSpan.FromSeconds(sources.Sum(source => source.Duration.TotalSeconds));
 
             if (!RunProcess(arguments, output, skipExistsCheck))
                 throw new FFMpegException(FFMpegExceptionType.Conversion, "Could not process file without error");
 
             _totalTime = TimeSpan.MinValue;
+
+            if (output == null)
+                return null;
             return new VideoInfo(output);
```

Comment:
I have already merged a fix for this from WeihanLi.
It is MUCH easier to review a PR if it is focused on one thing, instead of fixing unrelated stuff as well. Those fixes should be separate PRs, so they can be reviewed faster and merged sooner. Now this PR will have to wait until these lines have been reverted, to avoid conflict

Reply:
When i was writiting this code #50 was not merged. But without this line it would be impossible to run code at all

Reply:
Merged at ee16064